person.rb

```diff
@@ -6,6 +6,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Also there 👍. Kindly make sure a method for adding rentals is present 💪. Let's take an example form this:
```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  .
  .
end
```